packages/xod-client/src/editor/containers/Editor.jsx

```diff
@@ -47,26 +63,45 @@ class Editor extends React.Component {
       [COMMAND.SET_MODE_DEFAULT]: this.setModeDefault,
       [COMMAND.UNDO]: () => this.props.actions.undo(this.props.currentPatchPath),
       [COMMAND.REDO]: () => this.props.actions.redo(this.props.currentPatchPath),
+      [COMMAND.INSERT_NODE]: () => this.showSuggester(),
     };
   }
 
+  showSuggester(pos) {
+    this.props.actions.showSuggester(pos);
```

Comment:
A little bug: the suggester will be shown while editing a comment node and typing `i`.
I suggest checking here if event target is input or textarea.

Reply:
Actually, this reproduces with all kinds of inputs, not only comment editing.

Reply:
:open_mouth: nice catch!